build.sh

```diff
@@ -112,7 +112,7 @@ prepare_native_build()
     # Generate version.c if specified, else have an empty one.
     __versionSourceFile=$__scriptpath/bin/obj/version.c
     if [ $__generateversionsource == true ]; then
-        $__scriptpath/Tools/corerun $__scriptpath/Tools/MSBuild.exe "$__scriptpath/build.proj" /t:GenerateVersionSourceFile /p:NativeVersionSourceFile=$__scriptpath/bin/obj/version.c /p:GenerateVersionSourceFile=true /v:minimal
```

Comment:
That makes sense. Although to fix all that we will require an update in buildtools first. If it's ok with you Wes, I would like to get Joel unblocked and get this in, and then fix this in buildtools and update this scripts.